packages/server/graphql/types/Threadable.ts

```diff
@@ -67,8 +69,16 @@ const Threadable = new GraphQLInterfaceType({
   name: 'Threadable',
   description: 'An item that can be put in a thread',
   fields: threadableFields,
-  resolveType: ({status}) => {
-    return status ? Task : Comment
+  resolveType: (threadable: ThreadableDB) => {
```

Comment:
I'm not sure if `ThreadableDB` is the best name but there's nothing else that came to my mind. If you there's a better name to avoid collision with gql `Threadable` I'm happy to improve~

Reply:
i'm right there with ya, i've seen some people call it `CThreadable` where `C` is for `class`, but i think the DB suffix is a little more expressive